style.css

```
@@ -0,0 +1,142 @@
+html {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+
+body {
+    background-image: url(Images/sky.jpg);
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+
+}
+
+.container {
+    width: 100%;
+    min-width: 100vh;
+    padding: 1rem;
+
+
+}
+
+.todo {
+
+    width: 100%;
+    max-width: 34rem;
+    background-color: rgba(6, 0, 0, 0.346);
```

Fiind un fundal foarte încărcat, deranjează puțin vizual. Cred că ar merge puțin blur.

```suggestion
    background-color: rgba(6, 0, 0, 0.346);
    backdrop-filter: blur(8px); /* Aici sunt OK cu px */
```